docs/sources/datasources/azure-monitor/template-variables/index.md

```diff
@@ -32,16 +32,16 @@ For an introduction to templating and template variables, refer to the [Templati
 
 You can specify these Azure Monitor data source queries in the Variable edit view's **Query Type** field.
 
-| Name                | Description                                                                                  |
-| ------------------- | -------------------------------------------------------------------------------------------- |
-| **Subscriptions**   | Returns subscriptions.                                                                       |
-| **Resource Groups** | Returns resource groups for a specified subscription.                                        |
-| **Namespaces**      | Returns metric namespaces for the specified subscription and resource group.                 |
-| **Resource Names**  | Returns a list of resource names for a specified subscription, resource group and namespace. |
-| **Metric Names**    | Returns a list of metric names for a resource.                                               |
-| **Workspaces**      | Returns a list of workspaces for the specified subscription.                                 |
-| **Logs**            | Use a KQL query to return values.                                                            |
-| **Resource Graph**  | Use an ARG query to return values.                                                           |
+| Name                | Description                                                                                                        |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| **Subscriptions**   | Returns subscriptions.                                                                                             |
+| **Resource Groups** | Returns resource groups for a specified. Supports multi-value. subscription.                                       |
+| **Namespaces**      | Returns metric namespaces for the specified subscription and resource group.                                       |
+| **Resource Names**  | Returns a list of resource names for a specified subscription, resource group and namespace. Supports multi-value. |
+| **Metric Names**    | Returns a list of metric names for a resource.                                                                     |
+| **Workspaces**      | Returns a list of workspaces for the specified subscription.                                                       |
+| **Logs**            | Use a KQL query to return values.                                                                                  |
+| **Resource Graph**  | Use an ARG query to return values.                                                                                 |
```

Comment:
May be worth adding a reference to the new `Region` variable editor

Reply:
right! I forgot :)